cli/core/status.go

```diff
@@ -144,11 +146,18 @@ func GetStatus(ctx context.Context, eigenpodAddress string, eth *ethclient.Clien
 	proofSubmitter, err := eigenPod.ProofSubmitter(nil)
 	PanicOnError("failed to get eigenpod proof submitter", err)
 
-	currentOwnerShares, err := eigenPodManager.PodOwnerShares(nil, eigenPodOwner)
-	// currentOwnerShares = big.NewInt(0)
-	PanicOnError("failed to load pod owner shares", err)
-	currentOwnerSharesETH := IweiToEther(currentOwnerShares)
-	currentOwnerSharesWei := currentOwnerShares
+	delegationManagerAddress, err := eigenPodManager.DelegationManager(nil)
+	PanicOnError("failed to read delegationManager", err)
+
+	delegationManager, err := DelegationManager.NewDelegationManager(delegationManagerAddress, eth)
+	PanicOnError("failed to reach delegationManager", err)
+
+	_, shares, err := delegationManager.GetDepositedShares(nil, eigenPodOwner)
+	PanicOnError("failed to load owner shares", err)
+
+	// TODO: finish computation based on shares/strategies. (this is wrong)
+	currentOwnerSharesETH := IweiToEther(shares[0])
```

Comment:
@wadealexc - for the pod status, should we be using getDepositedShares directly in place of podOwnerShares? We probably also need to compute the sum of avs slashings that have occurred, right?

Reply:
(presumably this also the `sum(shares)` and not `shares[0]` :p )